app/assets/stylesheets/govuk-component/_breadcrumbs.scss

```diff
@@ -15,4 +17,30 @@
     color: $secondary-text-colour;
     text-decoration: none;
   }
+
+  @include media-down(mobile) {
+    &.collapse-on-mobile li {
+      display: none;
+
+      &.parent-breadcrumb {
+        background-image: none;
+        display: block;
+        margin-left: 0;
+        padding-left: 14px;
+        position: relative;
+
+        &:before {
+          border-bottom: 5px solid transparent;
+          border-right: 6px solid $back-link-arrow-color;
+          border-top: 5px solid transparent;
+          content: '';
+          display: block;
+          left: 0;
+          margin-top: -6px;
+          position: absolute;
```

Comment:
Feels like this should be possible without absolute positioning. Not sure without poking it but this would allow you to remove the magic positioning numbers.